### PR TITLE
Move list of wiki maintainers to `People`

### PR DESCRIPTION
**Pull request by Walavouchey**
had this on the to-do list for a while

updated title in languages i was confident i wouldn't run into grammar shenanigans

`people` and not `people/the_team` to follow pattern of no user group

### CODE REVIEW

File: wiki/Main_Page/de.md
Comment (RockRoller01) [inline wiki/Main_Page/de.md:119]:
```suggestion
[Community-Mitwirkende](/wiki/People/Community_Contributors) • [Benutzer mit einzigartigen Titeln](/wiki/People/Users_with_unique_titles) • [Turnierkomitee](/wiki/People/Tournament_Committee) • [Performance-Punkte-Komitee](/wiki/People/Performance_Points_Committee) • [Betreuer des osu!-Wikis](/wiki/people/osu!_wiki_maintainers)
```

File: wiki/People/de.md
Comment (RockRoller01) [inline wiki/People/de.md:6]:
```suggestion
- [Betreuer des osu!-Wikis](osu!_wiki_maintainers)
```

File: wiki/People/osu!_wiki_maintainers/de.md
Comment (RockRoller01) [inline wiki/People/osu!_wiki_maintainers/de.md:6]:
```suggestion
# Betreuer des osu!-Wikis
```

**Review by RockRoller01 — CHANGES_REQUESTED** [review]
Current TL is grammatically wrong, but that doesn't matter much as the chosen Noun is meh. The article itself uses "Betreuer", tho maybe "Instandthalter" would be better, but I leave that to the people that are more familiar with the german TL

**Comment by The-Last-Cookie** [comment]
`Betreuer` sounds fine to me.

The word `Instandhalter` was never used as far as I know.

All wiki articles use both `Betreuer` and `Verantwortlicher` for the translation of maintainer to have some word variety. Maybe `Leiter` is a good alternative too? :thinking:

Regardless of this, the changes that @RockRoller01 proposes can be merged into the branch in my opinion.

**Review by TicClick — APPROVED** [review]
tentative approval from me -- not sure how many changes you want to apply to the translations right now; I included mine, the rest (merge or not yet) is up to you

**Comment by Walavouchey** [comment]
i indeed wasn't intending on updating all of them myself, just the ones that seemed feasible to do quickly (and i only changed the titles)

**Pull request merged**
